source/includes/authentication/azure-imds-connection-string.py

```diff
@@ -1,9 +1,8 @@
 from pymongo import MongoClient
 
-# define properties, URI, and MongoClient
-properties = {"ENVIRONMENT": "azure", "TOKEN_RESOURCE": "<audience>"}
+# define URI and MongoClient
 uri = ("mongodb://<hostname>:<port>/?"
        "username=<Azure identity client ID>"
        "&authMechanism=MONGODB-OIDC"
-       "&authMechanismProperties=properties")
+       "&authMechanismProperties=environment:azure,token_resource:<audience>")
```

Comment:
ENVIRONMENT and TOKEN_RESOURCE should be capitalized by convention in all usages

Reply:
fixed